### ISSUE TITLE
$workspaceRoot is ignored during search of include files

### ISSUE THREAD

**Issue by dmazuronak** [microsoft/vscode-cpptools#72]
vscode marks all include files from the project directory as "not found", but the settings in c_cpp_properties.json includes the variable $workspaceRoot:

```
{
            "name": "Linux",
            "includePath": ["/usr/include", "/usr/include/c++/4.8",  "${workspaceRoot}"]
}
```

**Comment by sridmad** [comment]
All files under the opened folder should be processed without having to require the path in includePath.

Do you have a small repro for the issue of includes under root folder not processed?

**Comment by dmazuronak** [comment]
I've attached such [project](https://github.com/Microsoft/vscppsamples/files/317849/testproject.zip). First include is OK, but the second is marked as missing.

**Comment by jgoshi** [comment]
I wasn't able to repro this. It appears fixed with the latest release (version 0.7.0).

**Comment by cantide5ga** [comment]
@sridmad, @jgoshi  Should we expect variable substitution to work in `c_cpp_properties.json`?

My project(s) root as convention contains user-defined header files in an include folder.  I'd like to generically include this path for implementations and tests instead of using a relative path in the `#include`.
